src/client/log/log.js

```diff
@@ -34,12 +39,14 @@ const LogEvent = props => {
       onMouseLeave={() => props.onMouseLeave()}
     >
       {action.payload.type}({args.join(',')})
```

Comment:
Wrap this line and the next in a separate <div>:

```
<div>{ type and args }</div>
<div>{custompayload}</div>
```

That should make it better already. I'll take a look later when I'm at my computer and tweak the CSS.

Reply:
This does not work, it makes the payload not showing up at all. It gets shifted down below the next log event.

Reply:
Ah I see. OK, will take a closer look later and make some changes to your PR.

Reply:
... as always ;) thanks.